packages/utils/src/dom/html.ts

```diff
@@ -199,4 +222,12 @@ export class HTMLDocument {
 
         return result;
     }
+
+    public resolveUrl(url: string) {
+        if (!this._base) {
```

Comment:
This check shouldn't be necessary anymore.

Reply:
you are right!